cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -942,6 +942,43 @@ public void basicFT(String testcase)
     logger.fine("basicFT: " + testcase + " OK ");
   }
 
+  /**
+   * Isolated fulltext search tests for special characters.
+   *
+   * The use of a single equal operator '=' results in a fulltext search.
+   * Additional characters are tested to ensure that they do not cause any problems.
+   *.
+   * This uses www, xxx, yyy, and zzz to explicitly avoid potential stop-words.
+   *
+   * @param testcase conditions expected to pass.
+   */
+  @Test
+  @Parameters({
+    "text='text' # a; b; c; d; e; f",
+    "text='text without www' # a",
+    "text='text with' # b; c; d; e; f",
+    "text='text with xxx-yyy' # b",
+    "text='text with xxx-' # c",
+    "text='text with xxx' # c",
+    "text='text with yyy' # d",
+    "text='text with -yyy' # d",
```

Comment:
Thank you for the suggestion to add test cases where minus/hyphen is a prefix or suffix of a word. I've created https://github.com/folio-org/raml-module-builder/pull/607 for this.